src/buffer.rs

```diff
@@ -64,7 +64,7 @@ pub struct AudioBufferOptions {
 ///
 /// - `cargo run --release --example audio_buffer`
 ///
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
```

Comment:
This default implementation will create an audio buffer with zero channels, which we deem invalid - `assert_valid_number_of_channels` requires at least one.
This is a bit tricky, I'd rather not expose this, but there is no other way provide a default implementation that does not allocate..

Reply:
One solution would be to change
`channels: Vec<ChannelData>` to
`channels: SmallVec<[ChannelData; 2]>,`
which is probably a nice performance gain anyway.

Reply:
I have written up #334 in case you would like to defer this discussion and focus on the PR at hand

Reply:
Adding `Default` was just for convenience. I can create a tombstone instance manually that replaces the message content.